src/Shape/Whisker.js

```diff
@@ -0,0 +1,257 @@
+import {select} from "d3-selection";
+
+import {accessor, assign, BaseClass, configPrep, constant, elem} from "d3plus-common";
+
+import Circle from "./Circle";
+import Line from "./Line";
+import Rect from "./Rect";
+
+/**
+    @class Whisker
+    @extends BaseClass
+    @desc Creates SVG lines based on an array of data.
+*/
+export default class Whisker extends BaseClass {
+
+  /**
+      @memberof Whisker
+      @desc Invoked when creating a new class instance, and overrides any default parameters inherited from BaseClass.
+      @private
+  */
+  constructor() {
+
+    super();
+
+    this._endpoint = constant("Circle");
+    this._endpointConfig = {
+      Circle: {
+        fill: d => d.fill,
+        r: d => d.r
+      },
+      Rect: {
+        fill: d => d.fill,
+        height: d => d.height,
+        stroke: constant("black"),
+        strokeWidth: constant(1),
+        width: d => d.width
```

Comment:
We cannot have accessors in the default configs that reference user data that may not exist. These should all be static values.